### ISSUE TITLE
Redesign `/modules/[slug]` page for registry-v2

### ISSUE THREAD

**Issue by matifali** [coder/internal#190]
> The tabs I want to see on `/modules/[slug]` pages are:
> 1. README.md --> renamed to docs/documentation
> 2. Variables --> Renamed to Inputs
> 3. Outputs --> [NEW]
> 4. Parameters
> 5. Apps
> 6. Source --> [REMOVE] and link to GitHub 
> 
> Additionally, download count.
> 
> [![Image](https://github.com/user-attachments/assets/a4729c41-7416-4906-a6a9-0a6d9ed3c13e)](https://www.figma.com/design/j1geLGOj2tb70Orw9l4jrq/Registry-v2?node-id=0-1&node-type=canvas&t=B6N3Wv10J40pv36P-0)
> 
> What do you think @chrifro and @bpmct?
>  

 _Originally posted by @matifali in [#125](https://github.com/coder/internal/issues/125#issuecomment-2453788718)_

**Comment by matifali** [comment]
When you are doing the download count you may consider this @Parkreiner @bcpeinhardt.
It is. nice to have for now that we can do later too.

**Comment by Parkreiner** [comment]
@matifali Yeah, I definitely think that if push comes to shove, the download count can be demoted to a nice-to-have

I don't know if we have any data on how many people visit the current registry, but I think my main concern was that the current registry displays the most popular modules at the moment. Without the download counts, we'd have to showcase things alphabetically, and potentially show a bunch of modules that aren't really ever being used

**Comment by bcpeinhardt** [comment]
Since this is a new feature request I'm taking it off the stability milestone and we'll link it to wherever we track the new features we work on in the new year 😎

**Comment by matifali** [comment]
@chrifro to start thinking about the Registry design in general as we ramp up the registry going into 2025. I will let you know when this gets prioritized.

**Comment by matifali** [comment]
We now have a design for the page with a version picker. 
Let's prioritize this.

cc: @f0ssel

**Comment by buenos-nachos** [comment]
@matifali Is there a link to the design? I'm not sure if you mean the mockup that the contracting firm gave us, or an even more updated version?

**Comment by matifali** [comment]
@Parkreiner, I mean the Figma mockups we received from the Contractors. 

![Image](https://github.com/user-attachments/assets/24029a65-8550-47f7-b2a9-f3918ad8c78d)
